torch/csrc/jit/export.cpp

```diff
@@ -217,7 +217,7 @@ EncoderBase::EncoderBase(
   // we pin IR version to version 4 (01/22/2019) instead of using
   // onnx::IR_VERSION. with this change, the test_operators.py will be more
   // stable. only bump it when it's necessary
-  model_proto_.set_ir_version(4);
+  model_proto_.set_ir_version(6);
   // TODO: set the producer version using appropriate function call
   model_proto_.set_producer_version("1.3");
```

Comment:
Same comment about `producer_version` here.

Reply:
We should really unify two variables in py and c++ together :-)

Reply:
updated, also removed duplication by creating static info in torch/csrc/onnx/onnx.h